src/drivers/x11/lv_x11_input.c

```diff
@@ -10,12 +10,11 @@
 
 #if LV_USE_X11
 
+#include <string.h>
 #include <stdbool.h>
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
-#include "../../stdlib/lv_string.h"
```

Comment:
I think we should keep `lv_string.h` and use `lv_memmove` [here](https://github.com/lvgl/lvgl/blob/master/src/drivers/x11/lv_x11_input.c#L229).

Reply:
x11 lib probably needs those standard headers anyway, so I think it's OK to use `memmove` etc. in drivers/.

Reply:
I looked at what was done for the other platform-specific drivers. Standard headers seem to be used intentionally.
https://github.com/lvgl/lvgl/pull/5767/files#diff-f987953b6b5639df99192994498283ec69629b506874914605a14fd7afd12728

Reply:
Okay, let's keep the suggested changes.